test/sanitize-html.test.js

```diff
@@ -34,6 +34,13 @@ describe('Sanitize HTML', () => {
     );
   });
 
+  test('<iframe> gets removed after run through sanitizeHTML', () => {
+    const data = sanitizeHTML(
+      '<iframe src="https://www.telescope.com" style="border:none;">Telescope</iframe>'
+    );
+    expect(data).toBe('<iframe>Telescope</iframe>');
```

Comment:
So it hasn't stripped the `<iframe>` but rather, removed the `src` attribute.  Is that expected?  I would have thought it would remove the entire tag.  See my comment above.